docs/comparisons/README.md

```diff
@@ -0,0 +1,9 @@
+# Comparisons between .NET Standard and other platforms
+
+Many have asked for a diff between .NET Standard and other existing platforms.
```

Comment:
Maybe shortly explain how to read the diffs?
Does
```diff
+namespace Accessibility {
```
mean that .NET Standard 2.0 has that namespace or does it mean that .NET Framework 4.6.1 has that namespace?